app/models/manageiq/providers/azure/cloud_manager/event_catcher/runner.rb

```diff
@@ -40,7 +40,10 @@ def reset_event_monitor_handle
   end
 
   def filtered?(event)
-    event_type = parse_event_type(event)
+    event_type    = parse_event_type(event)
+    provider_type = event["resourceProviderName"]["value"]
+
+    @ems.blacklisted_provider_types.match(provider_type)||
```

Comment:
The spacing here is a bit funny.  Can you either put this `||` statement on one line, or indent the next line to make it clear it's part of the `||`.  Also, add a space between the `match` call and the `||`.